src/functions/JWT/generateJWT.js

```diff
@@ -1,6 +1,8 @@
 const jose = require("jose");
 
-module.exports = async (jti, sub, issuer, payload = {}, expiry = "1y") => {
+const jwtExpiry = process.env.JWT_EXPIRY_TIME || "1Y";
```

Comment:
@praveen-betalectic I think it should "1y" (small y), plz check.

Also, plz write with other values are valid, so that we can put in documentation.

Reply:
@rjvim i will check and update you.

Reply:
@rjvim 
- it is using standard timespan formats
- in standard timespan formats the difference between upper case letters and lowercase letters is not considered
- so `1Y` is same as `1y`
- other possible formats are 
   - "s" for seconds
   - "m" for minutes
   - "h" for hours
   - "d" for days
   - "w" for weeks
   - "y" for years
- and we can also use combinations like `1y2m10d4h34m20s`

> changed to lowercase 'y' for convention

Reply:
@rjvim changed to lowercase `y` , please review and merge.